plaso/parsers/sqlite_plugins/skype.py

```diff
@@ -446,7 +446,7 @@ def ParseAccountInformation(
 
     display_name = self._GetRowValue(query_hash, row, 'given_displayname')
     fullname = self._GetRowValue(query_hash, row, 'fullname')
-    username = '{0:s} <{1:s}>'.format(fullname, display_name)
+    username = '{0!s} <{1!s}>'.format(fullname, display_name)
```

Comment:
why the change, what other type can fullname, display_name be?

Reply:
They can be None.

Reply:
Seeing this is a user represented string I opt changing this logic to represent username as:
```
if fullname and display_name:
  username = '{0:s} <{1:s}>'.format(fullname, display_name)
elif fullname:
  username = fullname
else:
  username = display_name
```

Reply:
This changes the format of the field though, I'm not keen on doing this. Display name is already set as a separate attribute in the event data.

Reply:
But results like `"Name <None>"` look silly and are not useful IMHO

Reply:
I don't agree, I think it's more obvious to the user that there's something strange about that row.

Reply:
Ack, can we add a comment to move this to the formatter and let's chat how to represent values that are not set.